src/main/java/frc/robot/commands/arm/HoldArmCommand.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.arm;
+
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.Intake;
+import frc.robot.subsystems.ArmSubsystem;
+
+public class HoldArmCommand extends CommandBase {
+  /** Creates a new BackDriveCommand. */
+
+  private final ArmSubsystem armSubsystem;
+
+  public HoldArmCommand(ArmSubsystem armSubsystem) {
+    this.armSubsystem = armSubsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(armSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    moveUp();
+  }
+
+  private void moveUp() {
```

Comment:
this is not what is' doing. w'ere holding. rename this hold

Reply:
and then yeah finish implementing this command. so execute will run the motor at that speed it'll have to be the u[p direction since we're fighting gravity.